influxdb/src/main/java/org/influxdb/impl/InfluxDBResultMapperHelper.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package org.influxdb.impl;
```

Comment:
Why does this class live in that package?

Reply:
This has to do with some package-private methods the influxdb-java driver contains which are used. I can extract them and put them in another class if needed.